jupyter_scheduler/environments.py

```diff
@@ -26,7 +27,10 @@ def output_formats_mapping(self) -> Dict[str, str]:
 
 
 class CondaEnvironmentManager(EnvironmentManager):
-    """Provides list of system installed conda environments"""
+    """Provides list of system installed conda environments,
+    falls back to the python executable that jupyter lab is
+    running in, if conda is not installed or activated
+    """
```

Comment:
Would suggest the following change to the docstring:

> Provides a list of Conda environments. If Conda is not installed or activated, it defaults to providing exclusively the Python executable that JupyterLab is currently running in.

Reply:
Thanks, that sounds much better.